app/schema/api/v1/models.rb

```diff
@@ -27,6 +27,42 @@ def medical_history
       }
     end
 
+    def patient
+      super.tap { |d| d[:properties].delete(:deleted_at) }
```

Comment:
What happens when someone updates a deleted record using the V1 api?

Reply:
The updates go through the system as though nothing changed.

Only difference is the deleted records should not show up on the UI.